models/employee.py

```diff
@@ -78,6 +80,25 @@ def update_from_dict(self, d):
         self.department = d.get('department')
         self.meta_department = get_meta_department(self.department)
 
+    def get_gravatar(self):
+        """Creates gravatar URL from email address."""
+        email = '{user}@{domain}'.format(user=self.username, domain=config.DOMAIN)
```

Comment:
How about moving this to an email property, similar to how full_name works? Seems pretty useful. Please also add a small test in employee_test.py, which will make sure this line is tested as well.